pkg/addon/common.go

```diff
@@ -266,7 +266,7 @@ func CommonAgentInstallNamespaceFromDeploymentConfigFunc(
 
 		config, err := utils.GetDesiredAddOnDeploymentConfig(addon, adcgetter)
 		if err != nil {
-			log.Error(err, fmt.Sprintf("failed to get deployment config for addon %s: ", addon.Name))
+			log.Info(fmt.Sprintf("failed to get deployment config for addon %s: %s", addon.Name, err.Error()))
```

Comment:
I updated this one because an error with a trace is also logged following it by `sdk-go`, so there were too many traces being logged.

Reply:
...though now I'm second guessing it because the top-level trace here is probably more helpful...